docs/self-managed/operational-guides/multi-region/dual-region-ops.md

```diff
@@ -398,13 +397,13 @@ desired={<Eight viewBox="140 40 680 500" />}
 | **Camunda 8**            | A standalone region with a fully functional Camunda 8 setup, including Zeebe, Operate, Tasklist, and Elasticsearch. | Restore dual-region functionality by deploying Camunda 8 (Zeebe and Elasticsearch) to the newly restored region. |
 | **Operate and Tasklist** | Operate and Tasklist are operational in the standalone region.                                                      | Operate and Tasklist need to stay disabled to avoid interference during the database backup and restore process. |
 
-#### How to get there
+#### Procedure
 
-From your initial dual-region deployment, your base Helm values file `camunda-values.yml` in `aws/dual-region/kubernetes` should still be present.
+This procedure requires your Helm values file, `camunda-values.yml,` in `aws/dual-region/kubernetes,` used to deploy Dial-region Camunda clusters.
```

Comment:
@theburi the `dial` region remains ☎️.
The previous comment was about the spelling not the uppercase.
`dual-region`